central/detection/lifecycle/manager_impl.go

```diff
@@ -248,6 +253,26 @@ func (m *managerImpl) buildMapAndCheckBaseline(indicatorSlice []*storage.Process
 	}
 }
 
+func (m *managerImpl) SendBaselineToSensor(baseline *storage.ProcessBaseline) error {
+	err := m.connectionManager.SendMessage(baseline.GetKey().GetClusterId(), &central.MsgToSensor{
```

Comment:
nit:  add clusterID var to cut down on having to use `baseline.GetKey().GetClusterId()` over and over.

Reply:
Done